desk/app/tenna.hoon

```diff
@@ -14,11 +14,13 @@
   ==
 +$  card     card:agent:gall
 --
-%+  verb  |
-%-  agent:dbug
 =|  state-0
 =*  state  -
 ^-  agent:gall
+%+  verb  |
+%-  agent:dbug
+%-  %-  agent:vita-client
+      [& ~nodmyn-dosrux]
```

Comment:
the boolean here sets activity collection ON by default. the patp is the ship we want to send activity pokes to.